src/tests/base_kubeapi_test.py

```diff
@@ -197,18 +196,6 @@ def find_matching_node_and_return_name():
         log.info(f"patching agent {agent.ref} with hostname {hostname}")
         agent.patch(hostname=hostname)
 
-    @classmethod
```

Comment:
not sure if I wanted to make a <nit> critique of your decision to remove this 

as there are two places in base_test where it could be argued this is reused
https://github.com/openshift/assisted-test-infra/pull/1891/files#diff-21e688d6eb3b87727264f6934118765f035f5bf7504dff17b3ab9fb388f4d054R151

https://github.com/openshift/assisted-test-infra/pull/1891/files#diff-21e688d6eb3b87727264f6934118765f035f5bf7504dff17b3ab9fb388f4d054R423

but the added complexity of dealing with obtaining the returned single node IP justifies the duplication (I think?)

Reply:
Right. In the next PR I intend to do a couple of more changes, specifically removing all those occurrences of dynamic changes of terraform object.